extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/ImageSource.kt

```diff
@@ -17,12 +17,6 @@ import java.util.*
  *
  */
 class ImageSource(builder: Builder) : Source(builder.sourceId) {
-
-  init {
-    sourceProperties.putAll(builder.properties)
-    volatileSourceProperties.putAll(builder.volatileProperties)
-  }
```

Comment:
I guess that wasn't intentional?

Reply:
Hmmm true, that should be retained for Sources that are _not_ GeoJson. Thanks for noticing.